### PR TITLE
chore: codeowners updated

### PR DESCRIPTION
**Pull request by alejandroEsc**
**What type of PR is this?**

Chore

**What this PR does/ why we need it**:

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:
A bit more fine control over the code owners for different addons

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Checklist**

* [ ] The commit message explains the changes and why are needed.
* [ ] The code builds and passes lint/style checks locally.
* [ ] The relevant subset of integration tests pass locally.
* [ ] The core changes are covered by tests.
* [ ] The documentation is updated where needed.
* [ ] Manually tested upgrades from previous addon revisions for updated addons.

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -6,9 +6,16 @@
...
-# Prometheus
-addons/prometheus/* @mesosphere/sig-ksphere-observability 
+# Logging and Metrics stack
+addons/prometheus/* @mesosphere/sig-ksphere-observability @mesosphere/sig-ksphere-catalog
+addons/elasticsearch/* @mesosphere/sig-ksphere-observability @mesosphere/sig-ksphere-catalog
+addons/elasticsearch-curator/* @mesosphere/sig-ksphere-observability @mesosphere/sig-ksphere-catalog
+addons/elasticsearchexporter/* @mesosphere/sig-ksphere-observability @mesosphere/sig-ksphere-catalog
+addons/fluentbit/* @mesosphere/sig-ksphere-observability @mesosphere/sig-ksphere-catalog
+addons/kibana/* @mesosphere/sig-ksphere-observability @mesosphere/sig-ksphere-catalog
+addons/prometheusadapter/* @mesosphere/sig-ksphere-observability @mesosphere/sig-ksphere-catalog
```
Comment (samvantran) [inline CODEOWNERS:16]: nit: move this up the list under prometheus so they're grouped together?

File: .gitignore [hunk 2]
```diff
@@ -5,3 +5,4 @@ kind-logs-*/
 **/*.kube/
 /kubeaddons-tests
 kba-kubeconfig-*
+admin.conf
```
Comment (samvantran) [inline .gitignore:8]: nit: needs newline

**Review by samvantran — APPROVED** [review]
minor nits but rest LGTM

**Review by shaneutt — APPROVED** [review]

**Comment by alejandroEsc** [comment]
stopping aws test, this isn't needed.

**Review by joejulian — APPROVED** [review]

**Pull request merged**